plugins/polio/tasks/vaccine_authorizations_mail_alerts.py

```diff
@@ -35,41 +33,47 @@ def vaccine_authorizations_60_days_expiration_email_alert(vaccine_auths, mailing
         )
 
         if next_vaccine_auth and obj.expiration_date < next_vaccine_auth.expiration_date:
-            send_mail(
-                f"ALERT: Vaccine Authorization {obj} arrives to expiration date in 2 months",
-                f"""
-                ALERT, 
-
-                {obj.country} nOPV2 vaccines authorization date will expire on {obj.expiration_date}.
-                A new authorization is {next_vaccine_auth}, with an expiry date on {next_vaccine_auth.expiration_date}.
-                Please take appropriate action as needed. 
-                Link to the platform vaccine authorization page : {vaccine_dashboard_link}/dashboard/polio/vaccinemodule/nopv2authorisation/accountId/{account_id}/order/-current_expiration_date/pageSize/20/page/1"
-                RRT team
-                """,
-                settings.DEFAULT_FROM_EMAIL,
-                mailing_list,
-            )
-
-            mail_sent = True
+            try:
+                send_mail(
+                    f"ALERT: Vaccine Authorization {obj} arrives to expiration date in 2 months",
+                    f"""
+                    ALERT, 
+
+                    {obj.country} nOPV2 vaccines authorization date will expire on {obj.expiration_date}.
+                    A new authorization is {next_vaccine_auth}, with an expiry date on {next_vaccine_auth.expiration_date}.
+                    Please take appropriate action as needed. 
+                    Link to the platform vaccine authorization page : {vaccine_dashboard_link}/dashboard/polio/vaccinemodule/nopv2authorisation/accountId/{account_id}/order/-current_expiration_date/pageSize/20/page/1"
+                    RRT team
+                    """,
+                    settings.DEFAULT_FROM_EMAIL,
+                    mailing_list,
+                )
+
+            except Exception as e:
```

Comment:
I'd probably have catched a `smtplib.SMTPRecipientsRefused` (https://docs.python.org/3/library/smtplib.html#smtplib.SMTPRecipientsRefused) because it seems that that's what happened in the Sentry error and it has the advantage of being more specific.